rules/windows/defense_evasion_posh_compressed.toml

```diff
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2021/10/19"
+maturity = "production"
+updated_date = "2021/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of decompression and base64 decoding combined in PowerShell scripts, which Malware and Hacktools
+heavily use to deobfuscate payloads and load them directly in memory to bypass defenses.
+"""
+from = "now-9m"
```

Comment:
This could be a noisy one for prod tools and so could use an FP entry

Reply:
Done